index.html

```diff
@@ -3388,21 +3388,21 @@ <h3>Media Types</h3>
 (or a shortened version) in the media type may or may not imply a proof. The
 Working Group is seeking implementation experience related to the use of media
 types and the expectation related to whether or not a particular media type
-is expected to be secure with a particular proof.
-        </p>
-        <p class="issue" data-number="1065">
-The VCWG is currently debating whether all media types that start with
-`application/vc+` indicate an expectation around the securing mechanism or not.
-Working Group is seeking implementation experience related to the use of media
-types and the expectation related to whether or not a particular media type
 is expected to be secure with a particular proof.
         </p>
         <p>
 There is one media type associated with the core data model:
 `application/vc+ld+json`. Other specifications, such as [[?VC-JWT]],
 define other media types that contain transformation rules that allow for
 those other syntaxes to be transformed to the `application/vc+ld+json`
-media type.
+media type. Media types that allow for transformation into
+`application/vc+ld+json` SHOULD begin with `application/vc+`.
```

Comment:
I think this is good. I was wondering if we should say anything at all about this... the upside of not saying something is that other media types like `application/foo` could have a section about transformation into a VC w/o having to register two media types;  `application/foo` and  `application/vc+foo`... maybe the argument there is that the thing that's important is the transformation rules, not the media type? Does differentiating it via a new media type always help?

I'm fine w/ the text above for now, and it is a `SHOULD` so, people that are working on `application/foo` can choose to ignore it because they're concerned about media type proliferation. If they ignore the advice, my expectation is that we'd still add them to the VC Specifications Directory.

To be clear, this isn't a change request, just noting thoughts hoping others will engage in a discussion on the benefits/drawbacks of the specification requirement.

Reply:
I'm a bit concerned that talking this way about `application/vc+` suggests that there's a single way to deal with any and all such media types, though the way to deal with a media type is most simply determined by the outer edges of the media type -- e.g., `application/vc+ld+json` is most simply handled as `application/json`, not as `application/vc+`.

Reply:
This has been refactored in [0d5dbd9](https://github.com/w3c/vc-data-model/pull/1107/commits/0d5dbd985a84c27311cc86225a898904f115c1af)